src/app/apps/vacunacion/components/inscripcion.component.ts

```diff
@@ -0,0 +1,170 @@
+import { Observable } from 'rxjs';
+import * as enumerados from '../../../utils/enumerados';
+import { LocalidadService } from 'src/app/services/localidad.service';
+import { cache } from '@andes/shared';
+import { ProfesionService } from 'src/app/services/profesion.service';
+import * as moment from 'moment';
+import { InscripcionService } from '../services/inscripcion.service';
+import { GrupoPoblacionalService } from 'src/app/services/grupo-poblacional.service';
+import { PlexModalComponent } from '@andes/plex/src/lib/modal/modal.component';
+
+@Component({
+    selector: 'inscripcion',
+    templateUrl: './inscripcion.component.html'
+})
+export class InscripcionComponent implements OnInit {
+    @ViewChild('modal', { static: true }) modal: PlexModalComponent;
+
+    public resultado = null;
+    recaptcha: any = null;
+    public localidades$: Observable<any>;
+    public profesiones$: Observable<any>;
+    public opcionesGrupos$: Observable<any>;
+
+    public sexos: any[];
+    public relacionLaboral = [
+        { id: 'planta', label: 'Personal de planta' },
+        { id: 'eventual', label: 'Personal con contrato eventual' },
+        { id: 'terciarizado', label: 'Personal de empresas que brindan servicios terciarizados en ese establecimiento' },
+        { id: 'locacion', label: 'Personal contratado por locación de servicios (monotributista, factura AFIP)' },
+        { id: 'locacion', label: 'Personal autónomo que factura honorarios a través de colegios o instituciones similares' },
+        { id: 'otros', label: 'Otros' },
+    ];
+    public ciudadano: ICiudadano = {
+        id: null,
+        fechaRegistro: null,
+        documento: '',
+        nombre: '',
+        apellido: '',
+        tieneTramite: true,
+        nroTramite: '',
+        grupo: undefined,
+        sexo: null,
+        fechaNacimiento: null,
+        localidad: undefined,
+        telefono: '',
+        email: '',
+        cue: '',
+        alergia: false,
+        condicion: false,
+        enfermedad: false,
+        convaleciente: false,
+        aislamiento: false,
+        vacuna: false,
+        plasma: false,
+        amamantando: false,
+        embarazada: false,
+        profesion: '',
+        matricula: null,
+        establecimiento: '',
+        localidadEstablecimiento: undefined,
+        relacion: '',
+        estado: 'pendiente'
+    };
+
+    public relacion = null;
+    public sexo = null;
+    public seleccionTramite = true;
+    public fechaMaximaNacimiento;
+    public profesion;
+    public patronDocumento = /^[1-9]{1}[0-9]{4,7}$/;
+    public patronContactoNumerico = /^[0-9]{3,4}[0-9]{6}$/;
+
+    constructor(
+        private plex: Plex,
+        private localidadService: LocalidadService,
+        private profesionesService: ProfesionService,
+        private inscripcionService: InscripcionService,
+        private grupoPoblacionalService: GrupoPoblacionalService,
+    ) {
+        this.plex.updateTitle('VACUNACIÓN');
```

Comment:
Al ser un sitio público, un título más descriptivo va a ayudar a quien googlee
```suggestion
        this.plex.updateTitle('Inscripción a vacunación COVID-19 - Provincia del Neuquén');
```

Reply:
No va a figurar en Google, los robotos no lo pueden encontrar porque no tiene javascript.

Reply:
no había visto el coment de marian, igual lo cambié

Reply:
Creo que index cosas básicas así como el title, incluso seteado con JavaScript. De todos modos, mejor que el título sea más descriptivo para el usuario.